pom.xml

```diff
@@ -120,6 +120,19 @@
         <artifactId>logback-classic</artifactId>
         <version>1.2.2</version>
       </dependency>
+
+
+      <!-- Bean validation -->
+      <dependency>
+        <groupId>org.hibernate</groupId>
+        <artifactId>hibernate-validator</artifactId>
```

Comment:
do we really need a hibernate module to do validation? i thought this added things like hibernate precommit and prepersist